atom/browser/browser_win.cc

```diff
@@ -125,15 +125,47 @@ bool Browser::SetUserTasks(const std::vector<UserTask>& tasks) {
   return SUCCEEDED(destinations->CommitList());
 }
 
-bool Browser::RemoveAsDefaultProtocolClient(const std::string& protocol) {
-  if (protocol.empty())
-    return false;
+std::wstring protocolLaunchPath(std::string protocol, mate::Arguments* args) {
+  // Read in optional exe path arg
+  std::wstring exePath;
+  std::string rawExePath;
```

Comment:
Use `base::FilePath` instead of `std::wstring` when dealing with paths, and `base::string16` for other stuff you may want to use a `std::wstring` for (at least for passing read-only strings to Win32 APIs).